web-console/knikubevirt/vm-details/Resources/Resources.md

```diff
@@ -0,0 +1,40 @@
+# VM details - Devices-and-resources
+
+### Virtual machine
+
+## From VM details page
+![VM details - Overview tab - default](img/VM-2-0.jpg)
+Within the VM details > overview tab > overview section, a click on the ‘Devices & resources’ field will pop up the correlated modal.
+
+![Devices & resources modal - default](img/VM-3-0.png)
+The ‘Devices & resources’ modal currently contains only the ‘workload isolation’ option. This may expand in the future. The modal is titled this way, instead of  ‘Workload isolation’, in order to align with the implementation on the pod’s side.
+
+A sub-text below the title mentions that the scheduler will attempt to achieve the following settings. In case it is not possible, the VM will be scheduled in the best way it can (the same thing applies for pods, where this modal has more settings).
+
+The user selects ‘Isolate this workload…’ checkbox.
```

Comment:
This should be updated to reflect the current modal right?

Reply:
Yeah, I just dropped the ball with the MD...